### ISSUE TITLE
Issue trying to install kos

### ISSUE THREAD

**Issue by EimaMei** [KallistiOS/KallistiOS#77]
When trying to launch `./download.sh` in the `dc-chain` folder it outputs this error:
```
*** Downloader Utility for Sega Dreamcast Toolchains Maker (dc-chain) ***

Downloading Binutils 2.34...
curl: --continue-at and --remote-header-name cannot be combined
curl: try 'curl --help' for more information
```
Not sure what I did wrong or if it's a KOS issue

**Comment by EimaMei** [comment]
Update: I changed the forced installer to `wget` in `config.mk` and it worked for me. Not sure why curl doesn't work, maybe because I am using Manjaro? No real clue

**Comment by ljsebald** [comment]
Should be fixed in 807ea13be5d8f5a43b3b328c1f50505be4f482a8. The flag that triggered that error message probably shouldn't have ever been used.
